tests/forte/processors/data_augment/algorithms/typo_replacement_op_test.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for dictionary word replacement op.
+"""
+
+import unittest
+from forte.data.data_pack import DataPack
+from ft.onto.base_ontology import Token
+from forte.processors.data_augment.algorithms.typo_replacement_op import (
+    TypoReplacementOp,
+)
+
+
+class TestTypoReplacementOp(unittest.TestCase):
+    def setUp(self):
+        self.tyre = TypoReplacementOp(
+            configs={
+                "prob": 1.0,
+                'typo_generator': 'uniform',
+                }
```

Comment:
Use [black](https://black.readthedocs.io/en/stable/) to format all python sources, so that we have correct indent.